spaces/views.py

```diff
@@ -45,3 +65,183 @@ def get(self, request):
         if not results:
             return JsonResponse({"message" : "SPACE_DOES_NOT_EXIST"}, status = 400)
         return JsonResponse({"results" : results}, status = 200)
+
+
+class HostView(View):
+
+    @transaction.atomic
+    @login_required
+    def post(self, request):
+        try:
+            user  = request.user
+            hosts = Host.objects.filter(user=user)
+            host  = hosts.get() if hosts.exists() \
+                        else Host.objects.create(user=user) 
+            
+            data = json.loads(request.body)
+            place_type    = data["place_type"] 
+            sub_property  = data["sub_property"]
+            max_people    = data["max_people"]
+            bathroom      = data["bathroom"]
+            is_with_host  = data["is_with_host"]
+            
+            country  = data["country"]
+            state    = data["state"]
+            city     = data["city"]
+            street   = data["street"]
+            bedrooms = data["bedrooms"] # as nested dictionary
+            options  = data["options"]  # as list
+
+            options = [Option.objects.get(name=option) for option in options] 
+
+            main_address   = ' ,'.join((street, city, state, country)) 
+            address_detail = data["address_detail"] 
+
+            place_type   = PlaceType.objects.get(name=place_type)
+            sub_property = SubProperty.objects.get(name=sub_property)
```

Comment:
`place_type   = PlaceType.objects.get(name=place_type)`코드를 사용하지 않고
SubProperty 객체로만 접근해서 원하는 sub_property 데이터를 조회해보세요~
```suggestion
            sub_property = SubProperty.objects.
```